git-flow/README.md

```diff
@@ -0,0 +1,73 @@
+# GitFlow and Release Model for SmaRP
+
+
+## Branching System
+
+We use a [**GitFlow**](https://nvie.com/posts/a-successful-git-branching-model/) branching model, where the repository holds two main **branches** with an infinite lifetime:
+
+- [**`master`**](https://github.com/miraisolutions/SmaRP/tree/master): reflects the [**latest release**](https://github.com/miraisolutions/SmaRP/releases/latest) to production, i.e. the current version of the [live app](https://mirai-solutions.ch/gallery/smarp).
+- [**`develop`**](https://github.com/miraisolutions/SmaRP/tree/develop): collects all completed developments for the [**next release**](#release).
+
+The overall GitFlow branching system is described as follows
+
+- No work is committed and pushed directly to `master`, which is updated only as part of a [**release**](#release).
+- Small (maintenance) work can be done directly in `develop`, however meaningful pieces should be developed in a dedicated **_feature_ branch** created from `develop` and associated to a GitHub issue. By convention, the branch name is of the form `feature/<ID>-short-title`.
```

Comment:
```suggestion
- Small (maintenance) work can be done directly in `develop`, however meaningful pieces should be developed in a dedicated **_feature_ branch** created from `develop` and associated to a GitHub issue. By convention, the branch name is of the form `feature/<ID>-short-title`. The same concept applies for bug fixes, in which case the branch will be called `fix/<ID>-short-title`.
```

Reply:
I am actually wondering if this is actually a convention we want to keep. In GitFlow there is no practical distinction between developing in a feature branch a bug fix as opposed to different type of development, also because the line is not that sharp: you can see any small enhancement as fixing something not ideal, and fixing a non-critical bug as a small enhancement.

What matters more are the hotfixes that goes straight from and to master. Happy to discuss (@RolandASc?), which is the only distinction GitFlow typically does.